xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-send/xwiki-platform-mail-send-default/src/main/java/org/xwiki/mail/script/ScriptMimeMessage.java

```diff
@@ -88,7 +88,8 @@
      * @param mimeType the mime type of the content parameter
      * @param content the content to include in the mail. The type depends on the mimetype. For example for a mime
      *        type of {@code text/plain} or {@code text/html}, the content should be a String, for a mime type of
-     *        {@code xwiki/template} it should be a {@link org.xwiki.model.reference.DocumentReference}, etc.
+     *        {@code xwiki/template} it should be a {@link org.xwiki.model.reference.DocumentReference}, etc. Also 
+     *        directly accepts a BodyPart object.
```

Comment:
--> "Also accepts a {@link BodyPart} object, in which case, the mime type is not used."